### ISSUE TITLE
Error attempting to run after install

### ISSUE THREAD

**Issue by PMPman** [LiroyvH/signal-export#2]
Hello LiroyvH

After installation I issued the command:
./sigexport.py EXPORT

Which returned the following error:
Traceback (most recent call last):
  File "/Applications/OSINT-Tools/signal-export/./sigexport.py", line 11, in <module>
    import click
ModuleNotFoundError: No module named 'click'

I have a:
MacBook Pro
2.9 GHz 6-Core Intel Core i9
Intel UHD Graphics 630 1536 MB
32 GB 2400 MHz DDR4
Running MacOS Ventura 13.4.1 

Thoughts?

Al

**Comment by PMPman** [comment]
I was able to resolve the issue. It started with my deprecated Python version preventing installation fo click. Had to reinstall Homebrew first. Once remedied I had to run Brew Doctor to help fix broken links. I was then able to install Click. Then when I attempted to run ./sigexport.py EXPORT again I got an error that pysqlcipher3 was missing. When. attempted install of pysqlcipher3 I got an error that sqlcipher was missing. i install it with 'brew reinstall sqlcipher ' then ran 'brew info sqlcipher' which uncovered some variables that needed fixing. Once fixed I was successfully able to run 'pip3 install pysqlcipher3' without errors. And your application is now working like a charm!
